float64 x
float64 y
float64 z
float64 yaw
float64 ox
float64 oy
float64 oz
---
bool traj_ready
